sky/templates/aws-ray.yml.j2

```diff
@@ -118,12 +118,15 @@ setup_commands:
   # Line 'sudo grep ..': set the number of threads per process to unlimited to avoid ray job submit stucking issue when the number of running ray jobs increase.
   # Line 'mkdir -p ..': disable host key check
   # Line 'python3 -c ..': patch the buggy ray files and enable `-o allow_other` option for `goofys`
-  - sudo systemctl stop unattended-upgrades || true;
+  - function mylsof { p=$(for pid in /proc/{0..9}*; do i=$(basename "$pid"); for file in "$pid"/fd/*; do link=$(readlink -e "$file"); if [ "$link" = "$1" ]; then echo "$i"; fi; done; done); echo "$p"; };
```

Comment:
Does this reliably handle the apt problem?

Reply:
The `mylsof filename` works the same as `lsof filename` returning the pids that hold the file handle, according to the test on a newly launched GCP VM (`lsof /dev/null` vs `mylsof /dev/null`).